From 19b2598f8a52ba8af07eb4904788d0843130b094 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 27 Oct 2009 15:24:18 +0000
Subject: [PATCH] Revert "abuild: minor cleanup"

This commit seems to kill the entire repository

This reverts commit 46aed95754ebeb17a3a367b3b41d0b6424fd18d9.
---
 buildrepo.in |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/buildrepo.in b/buildrepo.in
index d719c34..c211789 100755
--- a/buildrepo.in
+++ b/buildrepo.in
@@ -22,9 +22,9 @@ usage() {
 
 
 listpackages() {
+	cd "$aportsdir/$1"
 	for i in */APKBUILD; do
-		cd "$aportsdir"/$1/${i%/*} 
-		abuild listpkg
+		APKBUILD=$i abuild listpkg
 	done
 }
 
-- 
1.6.5

